src/components/Footer/Footer.styles.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+import Image from "next/image";
+import Link from "next/link";
+
+export const Wrapper = styled.footer`
+  display: flex;
```

Comment:
Adding padding on the wrapper we won't need to add it on the LogoImage and References component

```suggestion
  display: flex;
  padding: 2rem;
  box-sizing: border-box;
```